nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-ui/src/main/java/Messages_cn.properties

```diff
@@ -0,0 +1,785 @@
+#-------------------------------------------------
```

Comment:
@ijokarumawak I have modified some errors，but now ，the master branch looks like it is not successful by building . And i did not modify anything about those error points .
Is that nifi own error?